sdk/core/azure-core/src/main/java/com/azure/core/implementation/logging/LoggingUtils.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.logging;
+
+import com.azure.core.util.CoreUtils;
+
+/**
+ * Contains utility methods for logging.
+ */
+public class LoggingUtils {
```

Comment:
Let's make this `final` and also add a private ctor.

Reply:
Should we enable https://checkstyle.sourceforge.io/apidocs/com/puppycrawl/tools/checkstyle/checks/design/HideUtilityClassConstructorCheck.html ?